lib/posix-futex/Config.uk

```diff
@@ -1,7 +1,7 @@
 menuconfig LIBPOSIX_FUTEX
 	bool "posix-futex: Futex"
 	select NOLIBC if !HAVE_LIBC
```

Comment:
Shouldn't this be

`select NOLIBC if !HAVE_LIBC` -> `select LIBNOLIBC if !HAVE_LIBC`

I know it's not a change in this PR 😄

Reply:
You're right. I added that in the last commit